docs/source/visual_overview.rst

```diff
@@ -1,7 +1,8 @@
 A Visual Overview of Projects
 =============================
 
-This will be a high level visual overview of project relationships (i.e.
-Thomas' graphic from the dev meeting.)
+**A high level visual overview of project relationships**
+
+
+.. image:: _static/_images/repos_map.svg
```

Comment:
I'd like this to be integrated into the list of projects (subprojects / doc-proj-categories) rather than a standalone page.